source/content/webops-dashboard.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Visualize Site Activity with the WebOps Dashboard
+description: Gain valuable insight to your site's development and collaboration with Pantheon's WebOps Dashboard.
+tags: [dashboard, webops, analytics]
+categories: [platform]
+contributors: [edwardangert]
+searchboost: 150
+---
+
+Pantheon's [WebOps](https://pantheon.io/webops) Dashboard combines metrics and data in one page of easy to understand visualization of site development.
+
+## How to Access the WebOps Dashboard
+
+From the [User](https://dashboard.pantheon.io) or Org Dashboard, any Site that is not [Sandbox](/create-sites#sandbox-sites) or [Frozen](/platform-considerations#inactive-site-freezing) has a **Track WebOps** button:
+
+![Screenshot of a User Dashboard with a Site showing the Track WebOps button](../images/dashboard/user-dash-track-webops.png)
+
+## What Does the WebOps Dashboard Show?
+
+![Screenshot of the WebOps Dashboard](../images/dashboard/webops-dashboard.png)
```

Comment:
I updated this screenshot after pushing another change to Anita Drupal, to show that different users are shown in the panel on the right. I also cut out the feedback item to make the image more evergreen, as I suspect that widget won't be around much longer past the beta stage.